Files/Views/PaneHolderPage.xaml

```diff
@@ -92,15 +92,32 @@
             <ColumnDefinition
                 x:Name="SidebarColumn"
                 Width="{x:Bind SidebarWidth, Mode=TwoWay}"
-                MinWidth="200"
-                MaxWidth="500" />
+                MinWidth="{ThemeResource NavigationViewCompactPaneLength}"
+                MaxWidth="{ThemeResource NavigationViewCompactPaneLength}" />
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
 
+        <VisualStateManager.VisualStateGroups>
+            <VisualStateGroup>
+                <VisualState>
+                    <VisualState.StateTriggers>
+                        <StateTrigger IsActive="{x:Bind IsSidebarOpen, Mode=OneWay}" />
+                    </VisualState.StateTriggers>
+
+                    <VisualState.Setters>
+                        <Setter Target="SidebarColumn.MinWidth" Value="250"/>
+                        <Setter Target="SidebarColumn.MaxWidth" Value="363"/>
```

Comment:
What's the number 363 from?

Reply:
There is a comment about it at [ViewModels/SettingsViewModel.cs#L107](https://github.com/files-community/Files/pull/3895/files#diff-fa2d499b2b9f90e35ad8ca2f4cb53b5924069d9c55d0dc4d6bf536b4779a63f8R107)

It is a magic number where the sidebar is at its maximum and the window its mimimum width and it results something like this 😅 

![image](https://user-images.githubusercontent.com/42123156/110383676-ec9d2400-805c-11eb-848b-392cdb00212e.png)

It can be changed to 360 or 350 of course to make it not so random...

Reply:
@nvi9 @yaichenbaum FYI, if we do go with this "magic" number, it should be created as a constant with a descriptive name and/or comment alongside it.

Reply:
@nvi9 Can you try 350?

Reply:
350:

![image](https://user-images.githubusercontent.com/42123156/110384842-7ac5da00-805e-11eb-9ca8-1c735fe61ed8.png)

Reply:
@nvi9 That looks good!